test/test-computed-properties.js

```diff
@@ -213,6 +213,19 @@ it('does correct NaN checking', () => {
   assert(el.count === count);
 });
 
+it('resets compute validity on initialization to catch upgrade edge cases with internal, computed properties', () => {
```

Comment:
Here's the straightforward test case. Honestly, this is the most convincing argument IMO — it's clear that this should work as this test suggests. I.e., `1 + 2` should indeed equal `3` 😂